__tests__/api.test.js

```diff
@@ -78,3 +79,22 @@ describe("/api/articles/:article_id", () => {
     expect(body.message).toBe("Bad Request");
   });
 });
+
+describe("api/articles", () => {
+  it("should return with status code 200 and an array of article objects", async () => {
+    const { body } = await request(app).get("/api/articles").expect(200);
+    expect(Array.isArray(body.articles)).toBe(true);
+  });
+  it("should sort the arrays in descending order", async () => {
+    const { body } = await request(app).get("/api/articles").expect(200);
+
+    expect(body.articles).toBeSorted({
+      key: "created_at",
+      descending: true,
+    });
+  });
```

Comment:
Please write a test where you actually checking the article objects. Use a flexible jest matcher like .toMatchObject or .objectContaining for that.

Also please make sure you are checking the 'comment_count' property as well.